tools/check_format.py

```diff
@@ -16,7 +16,8 @@
 
 EXCLUDED_PREFIXES = ("./generated/", "./thirdparty/", "./build", "./.git/", "./bazel-", "./.cache",
                      "./source/extensions/extensions_build_config.bzl",
-                     "./tools/testdata/check_format/", "./tools/pyformat/", "./api/wasm/")
+                     "./tools/testdata/check_format/", "./tools/pyformat/", "./api/wasm/",
+                     "./test/extensions/filters/http/wasm/")
```

Comment:
Why do we need this? I don't see anything obvious.

Reply:
Done

Reply:
Not done ;)

Reply:
Do we need this or not?

Reply:
ERROR: ./test/extensions/filters/http/wasm/test_data/grpc_call_cpp.cc:21: unexpected direct dependency on google.protobuf, use the definitions in common/protobuf/protobuf.h instead.

The wasm .cc files disobey envoy style in this regard.